scripts/west_commands/boards.py

```diff
@@ -55,9 +55,6 @@ def do_add_parser(self, parser_adder):
 
         # Remember to update west-completion.bash if you add or remove
         # flags
-        parser.add_argument('-f', '--format', default=default_fmt,
```

Comment:
`west boards --cmakeformat ...` does nothing, even though it appears in the help message.

Reply:
that's a tricky part, cause `west boards` actually has it's own method for printing boards.

The `--cmake-format` is provided directly in `list_boards.py` when script is called directly, however the boards extension command simply appends all args from list_boards to its own parser.

Will look into the best way to handle this.

Reply:
fixed.

`--cmakeformat` no longer present in `west boards -h`